### ISSUE TITLE
offline AI

### ISSUE THREAD

**Issue by egycno** [duelists-unite/YGOPro2#8]
When will the offline AI return?
Can an external offline AI be added to the game?

**Comment by raulbatist** [comment]
I don't think so because when had it it was like a non-stop error so I guess they passed this away and unfortunately won't be added. I guess so.
